dipy/reconst/mapmri.py

```diff
@@ -269,6 +270,65 @@ def odf(self, sphere, s=0):
         odf = np.dot(I_s, self._mapmri_coef)
         return odf
 
+    def rtpp(self):
+        r""" Calculates the analytical return to the plane probability (RTPP)
+        [1]_.
+
+        References
+        ----------
+        .. [1] Ozarslan E. et. al, "Mean apparent propagator (MAP) MRI: A novel
+        diffusion imaging method for mapping tissue microstructure",
+        NeuroImage, 2013.
+        """
+        Bm = self.model.Bm
+        rtpp = 0
+        const = 1 / (np.sqrt(2 * np.pi) * self.mu[0])
+        for i in range(self.ind_mat.shape[0]):
```

Comment:
I think yes, but is it that slow? I mean, ind_mat.shape[0] is usually 22 or 50 for full brain can take some time, but still less than the actual fitting. Are there guidelines for cythonizing code in Dipy right?